microsoft-edge/webview2/releasenotes.md

```diff
@@ -14,37 +14,51 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edg
 
 Release notes for [WebView2 SDK][WebView2NuGetGallery].  
 
+## 0.X.XXX
+
+[NuGet package][WebView2NuGetGallery0.X.XXX] | minimum Microsoft Edge version 8X.0.XXX.X.
+
+**Re-compile your app after updating the NuGet package.** 
+
+*   **Announcement:** WebView2 now supports WinForms and WPF on .NET Framework 4.6.2 or later and .NET Core 3.0 or later in the **pre-release package**
```

Comment:
```suggestion
*   Announcement: WebView2 now supports WinForms and WPF on .NET Framework 4.6.2 or later and .NET Core 3.0 or later in the pre-release package.
```

Reply:
explaining it is in the pre-release package is really important and we want developers to know this - this is very unqiue to their development experience

Reply:
For emphasis, the MS Style Guide says to use italics.